### PR TITLE
U4-2827 Allow dashboards via manifest

### PR DESCRIPTION
**Pull request by Shazwazza**
http://issues.umbraco.org/issue/U4-2827

### CODE REVIEW

File: src/Umbraco.Core/Manifest/ManifestParser.cs [hunk 1]
```diff
@@ -185,36 +185,43 @@ internal static IEnumerable<PackageManifest> CreateManifests(params string[] man
...
                 //validate the gridEditors section
-                var gridEditors = deserialized.Properties().Where(x => x.Name == "gridEditors").ToArray();
+                var gridEditors = deserialized.Properties().Where(x => x.Name == ManifestBuilder.GridEditorsKey).ToArray();
                 if (gridEditors.Length > 1)
                 {
                     throw new FormatException("The manifest is not formatted correctly contains more than one 'gridEditors' element");
                 }
 
+                //validate the dashboards section
+                var dashboards = deserialized.Properties().Where(x => x.Name == "dashboards").ToArray();
```
Comment (bjarnef) [inline src/Umbraco.Core/Manifest/ManifestParser.cs]: @Shazwazza I guess you can use the `ManifestBuilder.DashboardsKey` variable here as well 😃

**Comment by abjerner** [comment]
@Shazwazza @nul800sebastiaan Any reason this hasn't been merged yet? I remember seeing this to be due in 7.9 on the issue tracker, so I thought for sure it had already been merged.

**Comment by nul800sebastiaan** [comment]
@abjerner yeah it was rescheduled a few times, we never found the time to do a code review 🙈 

If you're up for giving it a look then that would be awesome! 👍

**Comment by abjerner** [comment]
@nul800sebastiaan I'll add it to my list ;)

**Comment by Shazwazza** [comment]
Only caveat with this is that currently the manifest dashboards don't support the security options ... we can add that in later though i'm sure.

**Comment by Shazwazza** [comment]
@warrenbuckley I've re-targeted now, will need to clean up the conflicts

**Pull request merged**